docs/src/pages/guides/mutations.md

```diff
@@ -179,6 +179,24 @@ mutate(todo, {
 })
 ```
 
```

Comment:
I would appreciate it if we could add another sub-heading here (level 3). That would make it easier to send that information around.